crates/yas_core/src/libraries/tick_bitmap.cairo

```diff
@@ -152,12 +152,12 @@ mod TickBitmap {
             tick / divisor
         } else if tick >= negative_lower_bound { //tick: [-256, -1]
             one_negative
-        } else {
-            let mut ans = IntegerTrait::<i32>::new((tick.mag / divisor.mag) - 1, true);
-            if (mod_i32(tick, divisor) == zero) {
-                ans = ans + IntegerTrait::<i32>::new(1, false);
+        } else { //[-887272, -257]
```

Comment:
remove `//[-887272, -257]`

Reply:
these comments are for a better understanding of the code. They add legibility

Reply:
you could add some additional text explaining what it is to add more context